### PR TITLE
1.11.0 release prep

### PR DESCRIPTION
**Pull request by chrisohaver**
Adds 1.11.0 release notes

### CODE REVIEW

**Comment by chrisohaver** [comment]
First bump. I am getting sed errors when making Makefile.doc. I suspect BSD vs Linux sed.

**Review by Tantalor93 — DISMISSED** [review]

File: coremain/version.go [hunk 1]
```diff
@@ -2,7 +2,7 @@ package coremain
 
 // Various CoreDNS constants.
 const (
-	CoreVersion = "1.10.1"
+	CoreVersion = "1.11.1"
```
Comment (SuperQ) [inline coremain/version.go]: ?

```suggestion
	CoreVersion = "1.11.0"
```

Comment (chrisohaver) [reply coremain/version.go]: thanks!

File: notes/coredns-1.11.0.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+## Noteworthy Changes
+
+* add support unix socket for GRPC (https://github.com/coredns/coredns/pull/5943)
+* plugin/forward: Continue waiting after receiving malformed responses (https://github.com/coredns/coredns/pull/6014)
+* plugin/dnssec: on delegation, sign DS or NSEC of no DS. (https://github.com/coredns/coredns/pull/5899)
+* plugin/kubernetes: expose client-go internal request metrics (https://github.com/coredns/coredns/pull/5991)
+* Prevent fail counter of a proxy overflows (https://github.com/coredns/coredns/pull/5990)
+* plugin/rewrite: Introduce cname target rewrite rule to rewrite plugin (https://github.com/coredns/coredns/pull/6004)
+* plugin/health: Poll localhost by default (https://github.com/coredns/coredns/pull/5934)
+* plugin/k8s_extenral: Supports fallthrough option (https://github.com/coredns/coredns/pull/5959)
```
Comment (mrueg) [inline notes/coredns-1.11.0.md]:
```suggestion
* plugin/k8s_external: Supports fallthrough option (https://github.com/coredns/coredns/pull/5959)
```

**Pull request merged**